### DISCUSSION TITLE
How to deploy to the cloud server?

### DISCUSSION THREAD

**Discussion by YoKONCy** [withastro/docs#3026] [Ideas]
How to deploy to the cloud server?  I executed npm run locally build, and then I got the dist folder, which contains the server folder and the client folder. Then I uploaded the dist folder to the cloud server, and then I started the entry.mjs in the server folder through pm2, but I still couldn't access it through ip plus port

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

I guess you fixed your issue by now so I'm closing this. The discussions are intended to discuss improvements to the documentation; for any support questions, feel free to join our [Discord server](https://astro.build/chat)!

If you think this is a mistake and that this discussion should remain open, please feel free to ping me, I'll take a second look at it.

Thank your for your contribution!
